### PR TITLE
[DOCS] Add session ID to highlighted fields section in alert details flyout

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/2066.

Preview:
- [Manage detection alerts | Overview](https://security-docs_2067.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-manage.html#view-alert-details): Added note about the Session ID field and updated image.

### CODE REVIEW

File: docs/detections/alerts-ui-manage.asciidoc [hunk 1]
```diff
@@ -109,6 +109,8 @@ The alert details flyout also lists the number and names of cases to which the a
 
 The *Highlighted Fields* section displays the most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.
 
+NOTE: The *Session ID* field shows the number of alerts generated on a Linux host in a session. To begin collecting session data, you must enable the *Include session data* setting on your Endpoint Security integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```
Comment (benironside) [inline docs/detections/alerts-ui-manage.asciidoc:112]:
```suggestion
NOTE: The *Session ID* field provides a unique ID for the Linux session. To collect it and other session data, you must enable the *Include session data* setting on your Endpoint Security integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```

**Review by benironside — COMMENTED** [review]
Image looks good, I left a correction for the text.

File: docs/detections/alerts-ui-manage.asciidoc [hunk 2]
```diff
@@ -109,6 +109,8 @@ The alert details flyout also lists the number and names of cases to which the a
 
 The *Highlighted Fields* section displays the most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.
 
+NOTE: The *Session ID* field provides a unique ID for the Linux session. To collect it and other session data, you must enable the *Include session data* setting on your Endpoint Security integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```
Comment (joepeeples) [inline docs/detections/alerts-ui-manage.asciidoc:112]: "the Linux session" is a little awkward since otherwise this doesn't seem to be talking about a specific session. Alternatively, something like "the Linux session that generated the alert" but that's a bit wordy.

Also li'l shared attribute fix:
```suggestion
NOTE: The *Session ID* field provides a unique ID for Linux sessions. To collect it and other session data, you must enable the *Include session data* setting on your {endpoint-cloud-sec} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```

**Review by joepeeples — COMMENTED** [review]
Left a couple small edits. Also super small issue, but the updated screenshot (`alert-details-flyout.png`) has a lot of empty space toward the bottom that could be cropped out fairly easily.

File: docs/detections/alerts-ui-manage.asciidoc [hunk 3]
```diff
@@ -109,6 +109,8 @@ The alert details flyout also lists the number and names of cases to which the a
 
 The *Highlighted Fields* section displays the most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.
 
+NOTE: The *Session ID* field provides a unique ID for Linux sessions. To collect it and other session data, you must enable the *Include session data* setting on your {endpoint-cloud-sec} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/detections/alerts-ui-manage.asciidoc:112]: Yeah, I think it's worth noting. I was actually trying to remember what the field name was so thank you for providing it here!

File: docs/detections/alerts-ui-manage.asciidoc [hunk 4]
```diff
@@ -109,6 +109,8 @@ The alert details flyout also lists the number and names of cases to which the a
...
+
 The *Alert prevalence* column shows the total number of alerts within the selected timeframe that have identical values. For example, an alert with an alert prevalence of 3 for the `host.name` field means three alerts with the same `host.name` value exist within the given timeframe. Alert prevalence data can help you investigate relationships with other alerts and gain more context about the event producing the alert.
```
Comment (janmonschke) [inline docs/detections/alerts-ui-manage.asciidoc:114]: It might be worth noting here that users can click on the alert prevalence count in order to investigate the alerts in timeline.

Comment (janmonschke) [reply docs/detections/alerts-ui-manage.asciidoc:114]: Ah sorry, I did not see that one :D Thanks so much!
